types/oboe/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for oboe v2.0.3
+// Type definitions for oboe v2.1.4
```

Comment:
```suggestion
// Type definitions for oboe 2.1
```
sorry, missed this yesterday, this is now preferred semver here:
https://github.com/DefinitelyTyped/DefinitelyTyped#how-do-definitely-typed-package-versions-relate-to-versions-of-the-corresponding-library